setup.py

```diff
@@ -21,10 +21,14 @@
     name='nb2kg',
     author='Jupyter Development Team',
     author_email='jupyter@googlegroups.com',
-    description='Extension for Jupyter Notebook 4.2.x to enable remote kernels hosted by Kernel Gateway',
-    long_description = '''See `the README <https://github.com/jupyter/kernel_gateway_demos/nb2kg>`_ for more information.
+    description='Extension for Jupyter Notebook 4.2.x to enable remote kernels hosted by Kernel Gateway or Enterprise Gateway',
+    long_description = '''\
+NB2KG is a Jupyter Notebook extension for versions >= 4.2 that enables remote kernels hosted 
+by `Jupyter Kernel Gateway <https://pypi.org/project/jupyter_kernel_gateway>`_ 
+or `Jupyter Enterprise Gateway <https://pypi.org/project/jupyter_enterprise_gateway>`_.  
+See `README <https://github.com/jupyter-incubator/nb2kg>`_ for more information.
 ''',
-    url='https://github.com/jupyter/kernel_gateway_demos/nb2kg',
+    url='https://github.com/jupyter-incubator/nb2kg',
     version=VERSION_NS['__version__'],
     license='BSD',
     platforms=['Jupyter Notebook 4.2.x'],
```

Comment:
Similar comments about including Notebook 5.x.... btw, platforms in this case can be used as notebook platform compared to OS like how we used in EG ?